src/modules/common/components/ScrollingSearchContainer.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import style from './ScrollingSearchContainer.css'
+import { animateScroll } from 'react-scroll/modules/index'
+import SearchInput from './SearchInput'
+
+export class ScrollingSearchContainer extends React.Component {
+  static propTypes = {
+    filterText: PropTypes.string.isRequired,
+    onFilterTextChange: PropTypes.func.isRequired,
+    stickyTop: PropTypes.number.isRequired,
+    spaceSearch: PropTypes.bool,
+    children: PropTypes.node
+  }
+
+  static defaultProps = {
+    stickyTop: 0
+  }
+
+  onClick = () => {
+    const scrollTop = document.documentElement.scrollTop
+    if (scrollTop < this.state.top) {
+      this.scroll()
+    }
+  }
+
+  scroll = () => {
+    animateScroll.scrollTo(this.state.top, {duration: 500})
+  }
+
+  onFilterTextChange = value => {
+    this.props.onFilterTextChange(value)
+    this.scroll()
+  }
+
+  setDimensions = node => {
```

Comment:
I think we should stick to es6 methods in classes and no lambda members.
So `setDimensions(node) {`
When you are at it you can look arount in the project for these kind of definions.

Reply:
But these have to be bound explicitly (when used with event handling) which is quite annoying